src/util/Decode.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * @module Decode
+ */
+
+import { Codec } from '@polkadot/types/types'
+
+/**
+ * Dummy comment needed for correct doc display, do not remove.
+ */
+
+/**
+ * Checks for a non-null byte in the Uint8Array representation
+ * of the codec. This is a pretty sound indicator that the
+ * codec does not hold an empty default value. The reverse inference
+ * does not apply! Valid values could be encoded only with 0 bytes.
+ *
+ * @param codec Codec type to check for non-empty bytes.
```

Comment:
The general jsdoc style in the SDK includes starting with _The_ or _A_ (e.g. _The codec type..._). We should keep this consistent.